setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='projects-edx-platform-extensions',
-    version='1.1.6',
+    version='1.1.7',
```

Comment:
Seems this is going to need another bump.

Reply:
I don't see `1.1.7` tag in this repository. I bumped it in edx-solutions/edx-platform#1398 only to avoid caching and to force reinstalling this package while testing. I think `1.1.7` should be fine.